local/start.sh

```diff
@@ -111,10 +114,10 @@ echo "Starting hippo job..."
 
 case "${OSTYPE}" in
   darwin*)
-    nomad run -var="os=osx" job/hippo.nomad
+    nomad run -var="os=osx" -var="HIPPO_ADMIN_USERNAME=admin" -var="HIPPO_ADMIN_PASSWORD=password" job/hippo.nomad
```

Comment:
```suggestion
    nomad run -var="os=osx" -var="admin_username=${HIPPO_ADMIN_USERNAME}" -var="admin_password=${HIPPO_ADMIN_PASSWORD}" job/hippo.nomad
```

(Same for line 120 below)

Alternatively, we could do without explicitly passing these through and up on lines 4-5 do something like:
```bash
NOMAD_VAR_admin_username="${HIPPO_ADMIN_USERNAME:=admin}"
NOMAD_VAR_admin_password="${HIPPO_ADMIN_USERNAME:=password}"
```

Reply:
Will do.

Reply:
> is there a reason not to do that with the other vars in the example here?

Probably no reason.  Could go the `NOMAD_VAR_*` route for any vars that use the same value regardless of os/arch/etc scenario...

Reply:
I think I prefer it.  It allows the variable assignments to live closer together, and the job running logic to be consolidated